### PR TITLE
[new release] ssh-agent-unix and ssh-agent (0.2.0)

### PR DESCRIPTION
**Pull request by reynir**
Ssh-agent

- Project page: <a href="https://github.com/reynir/ocaml-ssh-agent/">https://github.com/reynir/ocaml-ssh-agent/</a>

##### CHANGES:

* Switch to dune

* Implement OpenSSH RSA certificates

### CODE REVIEW

File: packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+description: """
+An ssh-agent protocol implementation in OCaml.
+
+For the Qubes OS unikernel using this library see [qubes-mirage-ssh-agent](https://github.com/reynir/qubes-mirage-ssh-agent)."""
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+authors: "Reynir Björnsson <reynir@reynir.dk>"
+license: "BSD-2-clause"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+depends: ["ocaml" "ssh-agent" "angstrom-unix" "faraday"]
```
Comment (kit-ty-kate) [inline packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam]:
```suggestion
depends: [
  "ocaml"
  "dune" {>= "1.0"}
  "ssh-agent" {= version}
  "angstrom-unix"
  "faraday"
]
```

File: packages/ssh-agent/ssh-agent.0.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Ssh-agent protocol parser and serialization implementation."
+maintainer: "Reynir Björnsson <reynir@reynir.dk>"
+authors: "Reynir Björnsson <reynir@reynir.dk>"
+license: "BSD-2-clause"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build & >= "1.0"}
```
Comment (kit-ty-kate) [inline packages/ssh-agent/ssh-agent.0.2.0/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune shouldn't be tagged as a build dependency (see #14266)

File: packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+authors: "Reynir Björnsson <reynir@reynir.dk>"
+license: "BSD-2-clause"
+homepage: "https://github.com/reynir/ocaml-ssh-agent/"
+bug-reports: "https://github.com/reynir/ocaml-ssh-agent/issues/"
+depends: [
+  "ocaml"
+  "dune" {>= "1.0"}
+  "ssh-agent" {= version}
+  "angstrom-unix"
+  "faraday"
```
Comment (kit-ty-kate) [inline packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam]:
```suggestion
  "faraday"
  "cmdliner"
```
According to this error message a dependency towards `cmdliner` is missing
```
#=== ERROR while compiling ssh-agent-unix.0.2.0 ===============================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/reynir/ocaml-ssh-agent/releases/download/v0.2.0/ssh-agent-v0.2.0.tbz)
# path                 ~/.opam/4.09/.opam-switch/build/ssh-agent-unix.0.2.0
# command              ~/.opam/4.09/bin/dune build -p ssh-agent-unix -j 2
# exit-code            1
# env-file             ~/.opam/log/ssh-agent-unix-23-612ba6.env
# output-file          ~/.opam/log/ssh-agent-unix-23-612ba6.out
### output ###
# File "cmd/dune", line 5, characters 60-68:
# 5 |  (libraries ssh-agent nocrypto nocrypto.unix ssh-agent-unix cmdliner))
#                                                                 ^^^^^^^^
# Error: Library "cmdliner" not found.
# Hint: try: dune external-lib-deps --missing -p ssh-agent-unix -j 2 @@default
```

Comment (reynir) [reply packages/ssh-agent-unix/ssh-agent-unix.0.2.0/opam]: Argh! I didn't actually intend to publish ssh-agent-unix. The executable using cmdliner is just an example. I'll fix things, and do a new PR.
Sorry about that, and thank you so much for reviewing!

**Pull request closed without merging**